less/navigation.less

```diff
@@ -1 +1,48 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+.nav-bar {
+
+  .simpleCenter();
+  position: fixed;
```

Comment:
Great job on the stretch goal here! 💯 The sticky top looks good and the opacity looks amazing 🔥